### PR TITLE
Add swiss newspaper `NZZ`

### PR DESCRIPTION
**Pull request by MaxDall**

### CODE REVIEW

File: src/fundus/publishers/ch/__init__.py [hunk 1]
```diff
@@ -17,3 +19,13 @@ class CH(metaclass=PublisherGroup):
...
         ],
     )
+
+    NZZ = Publisher(
+        name="Neue Zürcher Zeitung (NZZ)",
+        domain="https://www.nzz.ch/",
+        parser=NZZParser,
+        sources=[
+            NewsMap("https://www.nzz.ch/sitemap/news0.xml"),
+            Sitemap("https://www.nzz.ch/sitemap.xml", sitemap_filter=inverse(regex_filter(r"sitemap/[\d]{4}/[\d]{2}"))),
```
Comment (addie9800) [inline src/fundus/publishers/ch/__init__.py:29]: There would also be some available RSS Feeds: https://www.nzz.ch/information/rss-ld.105364

File: src/fundus/publishers/ch/nzz.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class NZZParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = CSSSelector("div.headline p.headline__lead")
+        _subheadline_selector = CSSSelector("div.article h2.subtitle, div.article h5.articlecomponent")
+        _paragraph_selector = CSSSelector(
```
Comment (addie9800) [inline src/fundus/publishers/ch/nzz.py:20]: This [article](https://www.nzz.ch/international/usa-das-wichtigste-zum-rennen-um-das-weisse-haus-ld.1716364) has a table of contents, that is parsed as well.

Comment (addie9800) [reply src/fundus/publishers/ch/nzz.py:20]: Also, as it seems this only occurs in English articles, but the last line with `Latest Articles` is parsed as well. See here: https://www.nzz.ch/english/us-election-2024-the-latest-polls-on-trump-and-biden-ld.1829123

**Review by addie9800 — CHANGES_REQUESTED** [review]
Nice addition 👍

File: src/fundus/publishers/ch/__init__.py (hunk 1 above)
Comment (MaxDall) [reply src/fundus/publishers/ch/__init__.py:29]: I would argue here that those are not necessary here, judging by the news map.

**Review by addie9800 — APPROVED** [review]

**Pull request merged**